### PR TITLE
removed outdated stub decorators

### PR DESCRIPTION
**Pull request by pondrejk**
https://bugzilla.redhat.com/show_bug.cgi?id=1279539 has been fixed but the previous way of skipping tests for private bugzillas (@stubbed('Unstub when BZ1279539 is fixed')) did not account for that, so several tests were unnecessarily skipped.

### CODE REVIEW

**Comment by svtkachenko** [comment]
Test results please?

**Comment by oshtaier** [comment]
Yeah. Don't want to disappoint you, but these stubs were added only for that purpose - to re-run all these tests and fix any issues once BZ is fixed. As it is tier4 - it can be pretty time consuming task

**Comment by pondrejk** [comment]
Understood, I'm running standalone automation to run those two long tests, meanwhile pasting results from the negative test, that is rather quick:

```
nosetests -v tests/foreman/cli/test_syncplan.py:SyncPlanTestCase.test_negative_synchronize_custom_product_current_sync_date -s
...
----------------------------------------------------------------------
Ran 1 test in 211.388s

OK

nosetests -v tests/foreman/api/test_syncplan.py:SyncPlanSynchronizeTestCase.test_negative_synchronize_custom_product_current_sync_date -s:

...
----------------------------------------------------------------------
Ran 1 test in 168.505s

OK

nosetests -v tests/foreman/ui/test_syncplan.py:SyncPlanTestCase.test_negative_synchronize_custom_product_current_sync_date 
Verify product won't get synced immediately after adding association ... ok

----------------------------------------------------------------------
Ran 1 test in 233.400s

OK
```


```

**Comment by pondrejk** [comment]
Since the positive tests wait on the first re-trigger of a sync plan after a start date, there's no need to set the start date as current (it is missed anyway). I set the start date in past (hour minus something), which reduces the length we need to wait for the test to execute. Test results for positive tests in cli and api:

```
tests/foreman/cli/test_syncplan.py::SyncPlanTestCase::test_positive_synchronize_custom_product_current_sync_date PASSED
tests/foreman/cli/test_syncplan.py::SyncPlanTestCase::test_positive_synchronize_rh_product_current_sync_date PASSED
```

```
tests/foreman/api/test_syncplan.py::SyncPlanSynchronizeTestCase::test_positive_synchronize_custom_product_current_sync_date PASSED
tests/foreman/api/test_syncplan.py::SyncPlanSynchronizeTestCase::test_positive_synchronize_rh_product_current_sync_date PASSED
```


UI tests have an additional problem, investigation in progress.

**Comment by pondrejk** [comment]
Additional changes made to the way the ui test was executed, result:
```
nosetests -v tests/foreman/ui/test_syncplan.py:SyncPlanTestCase.test_positive_synchronize_custom_product_current_sync_date
Create a sync plan with past datetime as a sync date, add a ... ok

----------------------------------------------------------------------
Ran 1 test in 78.222s

OK
```
Ready for review again, @svtkachenko @oshtaier can you please take a look?

File: tests/foreman/cli/test_syncplan.py [hunk 1]
```diff
@@ -417,26 +412,27 @@ def test_positive_synchronize_custom_product_current_sync_date(self):
...
+            ).strftime("%Y-%m-%d %H:%M:%S"),
         })
-        product = make_product({'organization-id': self.org['id']})
-        repo = make_repository({'product-id': product['id']})
         # Associate sync plan with product
         Product.set_sync_plan({
             'id': product['id'],
             'sync-plan-id': sync_plan['id'],
         })
-        # Wait half of expected time
```
Comment (oshtaier) [inline tests/foreman/cli/test_syncplan.py]: It is not right to remove delay at all as process can be started right away, but not finish till your first validation. That means you will not be able to catch 1279539.

File: tests/foreman/cli/test_syncplan.py [hunk 2]
```diff
@@ -400,11 +397,9 @@ def test_negative_synchronize_custom_product_current_sync_date(self):
             )
             # validate the error message once unstubbed (#3611)
 
-    # This Bugzilla bug is private. It is impossible to fetch info about it.
-    @stubbed('Unstub when BZ1279539 is fixed')
     @tier4
     def test_positive_synchronize_custom_product_current_sync_date(self):
-        """Create a sync plan with current datetime as a sync date, add a
+        """Create a sync plan with a past datetime as a sync date, add a
```
Comment (oshtaier) [inline tests/foreman/cli/test_syncplan.py]: Why did you decide that is expected behaviour? Even test case name point into word `current`

Comment (rplevka) [inline tests/foreman/cli/test_syncplan.py]: Current sync date (time) does not make sense on defining a SC for a simple reason. Creating takes time and one usually misses the trigger as creating takes time.
I would recommend getting rid of 'current' time test case and change it to the 'past' one.
Better test something than nothing.

File: tests/foreman/cli/test_syncplan.py (hunk 1 above)
Comment (pondrejk) [inline tests/foreman/cli/test_syncplan.py]: Hello, I did the rename and I added some delay in front of the first assert. But the real problem here is that the way we check the repo sync doesn't catch sync in progress. But I'd like to fix this in a separate PR.

**Comment by pondrejk** [comment]
Please also note the cherry pick  #4771

**Pull request merged**